src/pages/matching/loading.tsx

```diff
@@ -1,22 +1,50 @@
 import styled from '@emotion/styled';
-
 import Image from 'next/image';
 import { useRouter } from 'next/router';
-
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import type { NextPage } from 'next';
-
 import type { LoadingAnimationKey } from 'types/common';
+import type { MatchingInformation } from 'types/matching';
 import { loadingAnimation } from 'animation';
 import { Button } from 'components/common';
 import { PAGE_PATH } from 'constants/common';
+import { useRandomMatching } from 'contexts/RandomMatchingProvider';
+import { useAuthenticationState } from 'hooks/services/common/useAuthenticationState';
 
 const MatchingLoading: NextPage = () => {
   const router = useRouter();
 
+  const randomMatching = useRandomMatching();
+
+  const { user } = useAuthenticationState();
+
   const [isCancel, setIsCancel] = useState(false);
 
+  useEffect(() => {
+    if (user === undefined) return;
+    const { id: userId, username } = user;
+
+    void randomMatching.startMatching({
+      userInformation: { id: userId, username },
```

Comment:
p2; 
user 바로 전달해도 될 것 같네요!

```suggestion
    void randomMatching.startMatching({
      userInformation: user,
```

Reply:
수정하였습니다!

작업 커밋 : 1236ddf